### PR TITLE
Add Vulpkanin Species

### PR DESCRIPTION
**Pull request by PolarTundraa**
## About the PR
Ported from https://github.com/cosmatic-drift-14/cosmatic-drift/pull/214
Adds the Vulpkanin species to RMC14!

A big thank you to TsjipTsjip, PursuitInAshes, Boaz1111, Vermidia and Dutch-VanDerLinde for the help in making this PR possible. My first ever PR, so apologies in advance for any accompanying issues.

## Why / Balance
More species and customisation options! No balance, should be functionally identical to humans.

## Technical details
Xenonids call them 'canine hosts', felt fitting as their equivalent in felinids are referred to as 'feline hosts'.

## Media

Armor and helmets

image

Bursted 

image

Masks

image
image
image
image

(SEE COMMENT R.E OTHER MASKS)

image
image

## Requirements

- [X] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [X] I have added media to this PR or it does not require an ingame showcase.

**Changelog**

### CODE REVIEW

**Comment by MACMAN2003** [comment]
i don't think we need another generic species for annoying people to emote spam during briefing with.

**Comment by Rainbeon** [comment]
> i don't think we need another generic species for annoying people to emote spam during briefing with.

You are hardlimited on how many emotes you can even use in a time frame, and its not like adding more species makes more emote spammers suddenly spontaneously appear. People said the same exact shit for Avali before it was added, but I've seen none of that and the most emote "spamming" I actually see is human yawn, and moth noises still. And ofcourse just general gun equip sound spamming.

**Comment by MACMAN2003** [comment]
> You are hardlimited on how many emotes you can even use in a time frame, and its not like adding more species makes more emote spammers suddenly spontaneously appear. People said the same exact shit for Avali before it was added, but I've seen none of that and the most emote "spamming" I actually see is human yawn, and moth noises still. And ofcourse just general gun equip sound spamming.

that assumes if they are alone. there's a very high chance that the vulp players will team up with the felinid players to cause even more of an annoying ruckus at briefing.

**Comment by Rainbeon** [comment]
> > You are hardlimited on how many emotes you can even use in a time frame, and its not like adding more species makes more emote spammers suddenly spontaneously appear. People said the same exact shit for Avali before it was added, but I've seen none of that and the most emote "spamming" I actually see is human yawn, and moth noises still. And ofcourse just general gun equip sound spamming.
> 
> that assumes if they are alone. there's a very high chance that the vulp players will team up with the felinid players to cause even more of an annoying ruckus at briefing.

The group of people that plays all these races is generally the same overlap wise, you are not generating more players out of thin air that will engage in such behavior, it will be the same as always. A varied soundscape is also just less annoying than 1 billion Felinid "* Meows" over and over and over with no thought or variance (assuming such a thing even happens in the first place, separate races are less likely to ping pong off each other I've found, human yawns tend to lead to more yawns, moth noises tend to lead to more noises, especially the scream).

**Comment by PolarTundraa** [comment]
alright, passed all tests save for the last which I'm told is a github issue

File: Resources/Prototypes/_RMC14/Entities/Mobs/Species/vulpkanin.yml [hunk 1]
```diff
@@ -0,0 +1,101 @@
+- type: entity
+  save: false
+  parent: CMBaseMobSpeciesOrganic
+  id: CMMobVulpkanin
```
Comment (QuietlyWhisper) [inline Resources/Prototypes/_RMC14/Entities/Mobs/Species/vulpkanin.yml:4]: New content IDs should be RMC and not CM

File: Resources/Audio/_RMC14/Voice/Vulpkanin/attributions.yml
Comment (QuietlyWhisper) [inline Resources/Audio/_RMC14/Voice/Vulpkanin/attributions.yml:1]: We can't use NC assets, assets marked with NC will need to be removed.

**Review by QuietlyWhisper — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/_RMC14/Entities/Mobs/Species/vulpkanin.yml (hunk 1 above)
Comment (PolarTundraa) [reply Resources/Prototypes/_RMC14/Entities/Mobs/Species/vulpkanin.yml:4]: slightly confused on this, I can change it but I didn't see Avali or Felinids having that on their IDs?

**Comment by QuietlyWhisper** [comment]
Can you also include images of these with 4 directional where relevant
- Wearing armor and helmet
- Bursted
- Wearing masks

Also ensure that
- Speech and emotes work for the species
- Chat replacements work for the xenos
- Markings work, and are added/removed correctly with helmet-hiding features, and scout invisibility

File: Resources/Audio/_RMC14/Voice/Vulpkanin/attributions.yml
Comment (PolarTundraa) [reply Resources/Audio/_RMC14/Voice/Vulpkanin/attributions.yml:1]: NC assets have been removed

File: Resources/Prototypes/_RMC14/Entities/Mobs/Species/vulpkanin.yml (hunk 1 above)
Comment (QuietlyWhisper) [reply Resources/Prototypes/_RMC14/Entities/Mobs/Species/vulpkanin.yml:4]: Just removing the CM part all together is fine too

**Comment by QuietlyWhisper** [comment]
Oh and one last thing, for lore purposes, if you guys have some sort of lore like on the [wiki](https://wiki.rouny-ss14.com/wiki/Species), it would be highly appreciated for their announcement.
Here's some examples from it;
image
image

**Comment by snebl** [comment]
There is an open PR that fixes some clipping issues for lizards -> #4656
Another one could be made for Vulps in the future.

File: Resources/Prototypes/_RMC14/Entities/Mobs/Species/vulpkanin.yml (hunk 1 above)
Comment (PolarTundraa) [reply Resources/Prototypes/_RMC14/Entities/Mobs/Species/vulpkanin.yml]: ID has been changed to RMCMobVulpkanin

**Comment by PolarTundraa** [comment]
Requested changes have been made

I don't have any lore to go off in regards to the wiki

**Comment by Dutch-VanDerLinde** [comment]
I can fix gas masks not hiding snouts, it's because gas mask fits both on suit storage and snouts so

**Comment by snebl** [comment]
> Requested changes have been made

You forgot to Re-request review

**Comment by PolarTundraa** [comment]
Also updated Media in the main post with the requested screenshots of mask, armour and bursted directionals.

**Review by Dutch-VanDerLinde — CHANGES_REQUESTED** [review]
Parent these from the human sounds like the other species to remove copypaste

**Comment by PolarTundraa** [comment]
> Parent these from the human sounds like the other species to remove copypaste ( the emote sounds)

This should be cleaned up now, tested and emotes are still working

**Comment by Dutch-VanDerLinde** [comment]
> How are these going to work? Separate items?

Same way lizard specific sprites work on wizden

**Comment by mmails** [comment]
> > How are these going to work? Separate items?
> 
> Same way lizard specific sprites work on wizden 

You're completely right I have no idea how that slipped my mind

**Review by QuietlyWhisper — CHANGES_REQUESTED** [review]
Needs a coif sprite;
image
image

**Comment by PolarTundraa** [comment]
A coif sprite they recieve!

image

**Comment by PolarTundraa** [comment]
Linter test fail is probably unrelated? I'm seeing it on other PRs too.

image

File: Resources/Prototypes/_RMC14/Entities/Clothing/Masks/masks.yml [hunk 2]
```diff
@@ -42,6 +42,8 @@
     sprite: _RMC14/Objects/Clothing/Mask/sterile.rsi
   - type: Item
     storedRotation: -90
+  - type: HideLayerClothing
+    slots: []
```
Comment (QuietlyWhisper) [inline Resources/Prototypes/_RMC14/Entities/Clothing/Masks/masks.yml:46]: Drop these changes as they're not related to the pr

File: Resources/Prototypes/_RMC14/Entities/Clothing/Masks/masks.yml [hunk 3]
```diff
@@ -68,6 +70,11 @@
     coverage: MOUTH
   - type: Item
     storedRotation: -90
+  - type: HideLayerClothing # TODO port pulled down sprite from SS13
+    slots:
+    - Hair
+    - HeadTop
+    - HeadSide
```
Comment (QuietlyWhisper) [inline Resources/Prototypes/_RMC14/Entities/Clothing/Masks/masks.yml:77]: Drop these changes as they're not related to the pr
(it needs to be changed to support optional toggle)

File: Resources/Audio/Voice/Talk/attributions.yml
Comment (QuietlyWhisper) [inline Resources/Audio/Voice/Talk/attributions.yml:1]: Drop these changes as they're not related to the pr

File: Content.Server/_RMC14/Dropship/Weapon/DropshipWeaponSystem.cs
Comment (QuietlyWhisper) [inline Content.Server/_RMC14/Dropship/Weapon/DropshipWeaponSystem.cs:1]: Drop these changes as they're not related to the pr

**Review by QuietlyWhisper — CHANGES_REQUESTED** [review]

**Comment by PolarTundraa** [comment]
Pretty sure all the unrelated changes are handled.

New rebreather sprite for vulpkanin has been added: 

image

**Review by QuietlyWhisper — DISMISSED** [review]
This should be good to go

**Comment by DrSmugleaf** [comment]
> i don't think we need another generic species for annoying people to emote spam during briefing with.
> that assumes if they are alone. there's a very high chance that the vulp players will team up with the felinid players to cause even more of an annoying ruckus at briefing.
@MACMAN2003 If you have a valid concern you are free to express it, but please do not vomit pointless garbage like this on the GitHub.
It is neither constructive nor useful, "vulpkanin players are going to be annoying" is something you would get banned for saying in-game.
If you want to be able to mute emotes, implement https://github.com/RMC-14/RMC-14/pull/4138#pullrequestreview-2332093028 properly.

**Comment by KingOnBudget** [comment]
snarlll rarrll hıssss

**Review by DrSmugleaf — APPROVED** [review]
Thanks for the PR!

**Pull request merged**